packages/cli/src/config/config.ts

```diff
@@ -122,6 +123,11 @@ async function parseArguments(): Promise<CliArgs> {
       description:
         'Enable or disable logging of user prompts for telemetry. Overrides settings files.',
     })
+    .option('hide-footer', {
```

Comment:
rather than adding this to config can we instead add it to settings.json with a property added to settings.ts.
Generally settings.json is a bette place for options that are small UI tweaks than command line flags. If there is a use case where a user wants to frequently change between showing the footer and not showing the footer then I would be open to including it in config.ts